pages/linux/true.md

```diff
@@ -0,0 +1,7 @@
+# true
+
+> Does nothing, but returns a successful exit status code of 0.
```

Comment:
Let us remove "Does nothing, but".

Also, can we add a sub-heading mentioning - "Use this with the || operator to make a command always exit with 0" ?

Reply:
Can do!